api/Dockerfile

```diff
@@ -1,18 +1,17 @@
-FROM python:3.10-alpine
+FROM python:3.10-slim
 
-ARG VERSION="0.0.0-dev"
+ARG VALOR_VERSION="0.0.0-dev"
```

Comment:
what was the reason for this change? this also requires updating `.github/workflows/api-docker.yml` (line 68)

also since this doesn't seem related to this PR i prefer this be a separate one

Reply:
thanks for calling this out! this was leftover from when I was trying to debug the build errors using `python-alpine`; will revert this back